pkg/services/rendering/rendering.go

```diff
@@ -110,6 +113,23 @@ func (rs *RenderingService) RenderErrorImage(err error) (*RenderResult, error) {
 }
 
 func (rs *RenderingService) Render(ctx context.Context, opts Opts) (*RenderResult, error) {
+	startTime := time.Now()
+	result, err := rs.render(ctx, opts)
+	elapsedTime := time.Since(startTime).Milliseconds()
+	if status.Code(err) == codes.DeadlineExceeded {
+		metrics.MRenderingRequestTotal.WithLabelValues("cancelled").Inc()
```

Comment:
Which variant is better (UK vs US) "cancelled" or "canceled"? I see we use "cancelled" more often in the code.

Reply:
I'm bad at language. Google it
> So, which is correct: canceled or cancelled? Well, both are! Canceled is typically preferred in American English while cancelled is the standard in British English. Plus, it's OK to use cancelled in the US, too.

Searching all code is 38 results for each of them :smile: Canceled seems to be more common in backend since go's context package uses canceled so lets stick with that?

Reply:
Yes, think so.